### ISSUE TITLE
ButtonRectangle getTextView() nullpointer exception

### ISSUE THREAD

**Issue by Jdruwe** [navasmdc/MaterialDesignLibrary#261]
I just upgraded to the newest version and I get a nullpointer exception when I call getTextView() on my ButtonRectangle. What happened?

**Comment by jmvines** [comment]
me too

**Comment by Chostakovitch** [comment]
Same here. I can't resolve that.

**Comment by AlanVerbner** [comment]
I think it's beacause on this commit https://github.com/navasmdc/MaterialDesignLibrary/commit/1c574cfdb6fafe9f2376cc0400ed0782857e8882#diff-332a5defe7ad66da425918b00fce1925 a TextView was added to ButtonRectangle but getter is accessing parent TextView that is null. 

There are 2 pull requests about this bug but none of them were merged yet:
- https://github.com/navasmdc/MaterialDesignLibrary/pull/275
- https://github.com/navasmdc/MaterialDesignLibrary/pull/31

**Comment by naphthalene** [comment]
Original pull request has been up since November 2014 - is this getting fixed soon?

**Comment by pseudozach** [comment]
yeah I have a related issue as well. button.settext throws npe. merging the fix soon?

**Comment by naphthalene** [comment]
@pseudozach PR #275 works for me. Just check out that fork and build it, then add the generated aar to your libs folder and include it in build.gradle. Works as a temporary workaround

**Comment by pseudozach** [comment]
@naphthalene thanks, I switched to 1.4 build it doesnt have this issue. I have another problem tho, click on buttonrectangle is super slow. I'll go dig for this one now.

**Comment by cotfas** [comment]
I have the same problem

On 1.4 works perfectly but when I update to 1.5 I`m getting NullPointerEx on (ButtonRectangle)setText 

I need the 1.5 version because of a Switch component bug which it was fixed on 1.5...

Any news ?

**Comment by cotfas** [comment]
Actually the Switch bug it doesn`t work on the 1.5 version as well.. :(
